packages/utilities/src/dom.ts

```diff
@@ -204,6 +206,27 @@ export function getRect(element: HTMLElement | Window | null): IRectangle | unde
   return rect;
 }
 
+/**
+ * Identify element as a portal by setting an attribute.
+ * @param element Element to mark as a portal.
+ */
+export function setPortalAttribute(element: HTMLElement): void {
+  element.setAttribute(dataPortalAttribute, 'true');
+}
+
+/**
+ * Determine whether a target is within a portal from perspective of root or optional parent.
+ * If parent is not in target's hierarchy, parent defaults to root.
+ * If both parent and child are within the same portal this function will return false.
+ */
+export function portalContainsElement(target: HTMLElement, parent?: HTMLElement): boolean {
```

Comment:
How can we surface this better than just burried inside a comment & this bit of code? I'm wondering if we detect this case in dev mode, etc? Also, having to do a recursive find on mousemove seems it might cause some perf issues down the road for the consumers.

Reply:
Surface what exactly? The existence of this function? I'm not sure, I feel like it's in the same category as other functions in this file. Are they surfaced elsewhere? It's called out in the PR description and I was going to send an email out around merge so that people know about the portals change in general and this utility helper.

How this is called is up to the caller, of course, but in this PR it's only called on `mouseenter` events specifically against `TooltipHost`, not on `mousemove` events. As with other functions in this file, I feel people need to be judicious in its use and provide parents to limit search. If it's still a concern do you have any other suggestions for implementation?

Reply:
I think we should think about moving this to a wiki / doc site later.